client/src/store/utility.js

```diff
@@ -0,0 +1,6 @@
+export const updateObject = (oldObject, updatedProperties) => {
+    return {
+        ...oldObject,
+        ...updatedProperties
+    };
+};
```

Comment:
Would suggest using the `immutable-js`    
This won't update the deeply nested objects which might come in the future.
Or you can use `redux-starter-kit` from [here](https://redux.js.org/recipes/structuring-reducers/immutable-update-patterns).

Read about immutable js from [here](https://immutable-js.github.io/immutable-js/)